src/main/java/com/sprint/mission/discodeit/controller/MessageController.java

```diff
@@ -0,0 +1,84 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.BinaryContentCreateRequest;
+import com.sprint.mission.discodeit.dto.MessageCreateRequest;
+import com.sprint.mission.discodeit.dto.MessageUpdateRequest;
+import com.sprint.mission.discodeit.entity.Message;
+import com.sprint.mission.discodeit.service.MessageService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Controller
+@RequiredArgsConstructor
+@RequestMapping("/api/message")
+public class MessageController {
+    private final MessageService messageService;
+
+    public List<BinaryContentCreateRequest> resolveAttachmentFile(List<MultipartFile> attachmentFile) {
+        return attachmentFile.stream()
+                .map(file -> {
+                    try {
+                    return new BinaryContentCreateRequest(
+                            file.getOriginalFilename(),
+                            file.getContentType(),
+                            file.getBytes()
+                    );
+                } catch (IOException e) {
+                        throw new RuntimeException(e);
+                    }
+        })
+                .collect(Collectors.toList());
```

Comment:
`.collect(Collectors.toList());`는 불변이 아니라서 `.toList()`를 사용해 불변으로 만드는 습관 가져가면 좋을 것 같아요.    
```suggestion
                .toList();
```